src/main/steps/c100-rebuild/consent-order/upload-confirmation/template.njk

```diff
@@ -0,0 +1,31 @@
+{% extends "common/template.njk" %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/notification-banner/macro.njk" import govukNotificationBanner %}
+{% from "govuk/components/warning-text/macro.njk" import govukWarningText %}
+{% from "govuk/components/inset-text/macro.njk" import govukInsetText %}
```

Comment:
@ShakiraKP Are we using these imported macros, govukButton, govukWarningText & govukInsetText ?